packages/jest-enzyme/src/index.js

```diff
@@ -3,7 +3,7 @@
  * This source code is licensed under the MIT-style license found in the
  * LICENSE file in the root directory of this source tree. *
  *
- * @providesModule setupTestFrameworkScriptFile
+ * @providesModule setupFilesAfterEnv
```

Comment:
Probably these directives are redundant 🤔
> @providesModule is a facebookism. I do not recommend that you use it, since all of your infra has to support it. But you can find out more about it at https://github.com/facebookarchive/node-haste and https://github.com/facebook/jest/tree/master/packages/jest-haste-map.
https://github.com/facebook/flow/issues/2648#issuecomment-254883663

Reply:
OK, thanks!